### PR TITLE
extract methods for de/quantize, prerequisite for implementing child-…

### PR DESCRIPTION
**Pull request by thiloSchlemmer**
Hi Loic, 

as anounced (some weeks ago) here the first refactorings. I'm unsure about the correct-naming with all the self._workingUnit*-stuff, but think it is a good starting point.

### CODE REVIEW

File: quantized_mesh_tile/terrain.py [hunk 1]
```diff
@@ -529,6 +532,52 @@ def toFile(self, filePath, gzipped=False):
...
+
+    def _getWorkingUnitLongitude(self):
+        if not self._workingUnitLongitude:
+            self._workingUnitLongitude = old_div(self.MAX, (self._east - self._west))
+        return self._workingUnitLongitude
+
+    def _getDeltaHeight(self):
+        if not self._deltaHeight:
+            max = self.header['maximumHeight']
+            min = self.header['minimumHeight']
```
Comment (loicgasser) [inline quantized_mesh_tile/terrain.py]: min and max are reserved in python, maybe adapt it

File: quantized_mesh_tile/terrain.py [hunk 2]
```diff
@@ -529,6 +532,52 @@ def toFile(self, filePath, gzipped=False):
...
+            self._deltaHeight = max - min
+        return self._deltaHeight
+
+    def _quantizeLatitude(self, latitude):
+        v = int(round((latitude - self._south) * self._getWorkingUnitLatitude()))
```
Comment (loicgasser) [inline quantized_mesh_tile/terrain.py]: just `return int(round((latitude - self._south) * self._getWorkingUnitLatitude()))`

File: quantized_mesh_tile/terrain.py [hunk 3]
```diff
@@ -529,6 +532,52 @@ def toFile(self, filePath, gzipped=False):
...
+        return v
+
+    def _quantizeLongitude(self, longitude):
+        u = int(round((longitude - self._west) * self._getWorkingUnitLongitude()))
```
Comment (loicgasser) [inline quantized_mesh_tile/terrain.py]: just `return int(round((longitude - self._west) * self._getWorkingUnitLongitude()))`

File: quantized_mesh_tile/terrain.py (hunk 1 above)
Comment (thiloSchlemmer) [reply quantized_mesh_tile/terrain.py]: ok

File: quantized_mesh_tile/terrain.py (hunk 2 above)
Comment (thiloSchlemmer) [reply quantized_mesh_tile/terrain.py]: ok

File: quantized_mesh_tile/terrain.py (hunk 3 above)
Comment (thiloSchlemmer) [reply quantized_mesh_tile/terrain.py]: ok

**Pull request merged**